android/assets/jsons/translations/Dutch.properties

```diff
@@ -802,12 +802,12 @@ Avoid [terrain] = Vermijd [terrain]
 # Maya calendar popup
 
  # Requires translation!
-The Mayan Long Count = 
+The Mayan Long Count = De Maya Long Count
  # Requires translation!
-Your scientists and theologians have devised a systematic approach to measuring long time spans - the Long Count. During the festivities whenever the current b'ak'tun ends, a Great Person will join you. = 
+Your scientists and theologians have devised a systematic approach to measuring long time spans - the Long Count. During the festivities whenever the current b'ak'tun ends, a Great Person will join you. = Je wetenschappers en theologen hebben een systematische benadering bedacht om lange tijdspannes te meten - de Long Count. Tijdens de festiviteiten wanneer de huidige b'ak'tun eindigt, zal een groot persoon bij u aansluiten.
```

Comment:
[klein dingetje] Je spreekt hier de speler met zowel 'u' als 'jij' aan, wat wat inconsistent is. Volgens mij gebruikt de rest van de vertaling voornamelijk je/jij als ik me goed herinner